packages/charts/package.json

```diff
@@ -34,7 +34,7 @@
     "recharts": "2.9.2"
   },
   "peerDependencies": {
-    "@ui5/webcomponents-react": "~1.21.0",
+    "@ui5/webcomponents-react": "~1.22.0",
     "@ui5/webcomponents-react-base": "~1.20.0",
```

Comment:
```suggestion
    "@ui5/webcomponents-react-base": "~1.22.0",
```

Reply:
Do we have changes in the `base` package? Otherwise the version won't be bumped, right?

Reply:
yes: https://github.com/SAP/ui5-webcomponents-react/pull/5205/files#diff-0a2e875708bc7e289f4845daec445fab53f5e136388cf365c6b7f2f7ea49b003